CLDR-18855 Clarify regional locale formatting expectations (#4908)
unicode-org/cldr: docs/site/translation/number-currency-formats/number-symbols.md

The following symbols are used to format numbers. The Approved symbols for the locale will be substituted for the placeholders described in [Number Patterns](/translation/number-currency-formats/number-and-currency-patterns). 

> For regional locales (e.g. en\_DE) where the language is not the primary language of the country/region (e.g. en\_DE; English as used in Germany), the number formats and date formats should follow the **formatting usage** as commonly used by the people who live in that country/region. This can often be difficult to determine with languages such as English that are operating as a lingua franca. In these situations, much of the available content is written for an international audience and the majority of the readers using the language are from other countries/regions.

Reviewer: ```suggestion
For regional locales where the language is not the primary language of the country/region (e.g. en\_DE, English as used in Germany; or es_SE, Spanish as used in Sweden), the number formats and date formats should follow the formatting usage **of that language** as commonly used by the people who live in that country/region. This can be difficult to determine with languages that are operating as a lingua franca by people not using it as their first language. In these situations, much of the available content is written for an international audience and most of the readers are from other countries/regions.
```

> For regional locales where the language is not the primary language of the country/region (e.g. en\_DE, English as used in Germany; or es_SE, Spanish as used in Sweden), the number formats and date formats should follow the formatting usage **of that language** as commonly used by the people who live in that country/region. This can be difficult to determine with languages that are operating as a lingua franca by people not using it as their first language. In these situations, much of the available content is written for an international audience and most of the readers are from other countries/regions.

Reviewer 2: ```suggestion
For regional locales where the language is not the primary language of the country/region (e.g. en\_DE, English as used in Germany; or es_SE, Spanish as used in Sweden), there are different possible options for the number formats and date formats.
As with other CLDR data, the most customary form of _that_ language as used in _that_ country should be followed.
This is generally one of two options (or something in between):

1. The conventions of one of the main regional variants (eg, en\_US or en\_GB), such as 1,234.5 or 21-02-2025 or February 21 2025
2. The conventions of one of the main languages used in the country, such as 1’234,5 or 2025-21-01 or 21 February 2025

The choice of which option is most customary can be difficult to determine with languages that are operating as a lingua franca for people who don't have it as their first language.
In these situations, much of the available content is written for an international audience and most of the readers are from other countries/regions.

Note that developers can choose a specific language variant, such as en\_GB, with a specific region's formatting by using [the Region Override U extension](https://www.unicode.org/reports/tr35/#RegionOverride) if they, for example, prefer British English formatting in a region/country with different number or date formatting.
```
Note: by putting line-breaks between sentences it is easier to see the differences in wording in the future. As long as there aren't blank lines, they'll form one paragraph.

> Developers can choose a specific language variant such as en\_GB or en\_US with a specific region's formatting by using [the Region Override U extension](https://www.unicode.org/reports/tr35/#RegionOverride) if they prefer British or American English in a region/country with different number or date formatting.

Reviewer: ```suggestion
Developers can choose a specific language variant, such as en\_GB, with a specific region's formatting by using [the Region Override U extension](https://www.unicode.org/reports/tr35/#RegionOverride) if they, for example, prefer British English formatting in a region/country with different number or date formatting.
```
